r-package/grf/R/forest_summary.R

```diff
@@ -28,30 +28,47 @@
 #'
 #' @export
 test_calibration = function(forest) {
```

Comment:
Should we make a note here about clusters, as we did for the other methods?

Reply:
I think we also need a bigger PR where we explain how to use the `test_calibration` function. Maybe we can work on that discussion then? The way we handle clusters here is not consistent with the rest of the package (i.e., 1 cluster = 1 vote).